### PR TITLE
Misc. Fixes

### PR DESCRIPTION
**Pull request by bwalsh**
This PR contains several fixes:
* missing vertexes created by ccle drug response now have 'drug_response' prefix

### CODE REVIEW

File: transform/ccle/drug_response.py [hunk 1]
```diff
@@ -153,6 +153,7 @@ def transform(biosample_path='outputs/ccle/Biosample.Vertex.json*',
         )
 
     emitter.close()
+    emitter.close()
```
Comment (kellrott) [inline transform/ccle/drug_response.py]: A second `.close()` call?

Comment (bwalsh) [reply transform/ccle/drug_response.py]: thanks.

**Comment by bwalsh** [comment]
also #241

File: src/bmeg/enrichers/transcript_enricher.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    # {
+    #   "db_type":"core",
+    #   "start":55287849,
+    #   "Parent":"ENSG00000115355",
+    #   "end":55296455,
+    #   "object_type":"Transcript",
+    #   "logic_name":"havana",
+    #   "assembly_name":"GRCh38",
+    #   "species":"homo_sapiens",
+    #   "biotype":"protein_coding",
```
[the remark this answers is not in the document]
Comment (bwalsh) [reply src/bmeg/enrichers/transcript_enricher.py]: yes.  I’d like to get master updated and then tackle that as a new PR

**Pull request merged**